models/Referral.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const ReferralSchema = new Schema(
+  {
+    tx_ref: {
+      type: String,
+      required: true,
+      unique: true
+    },
+    campaign: {
+      type: String,
+      enum: ["Support A Learner", "Edustipend at 2"]
```

Comment:
@Chiemezuo, thanks for adding this but this means we have to force the front-end to pass exactly this same enum value.
So in cases where the `utm_source` has no value, should this break?
I think it might be okay just making it a string not an `enum`.

Reply:
Yeahh, you remember that in the message you specified two possible values for it. 
I put it as an Enum for uniformity, and so that queries to search for what each campaign has generated can be easily done. 

But in a way, if you look at it, all donations will either be "Support A Learner" or something else. 😅

I can take it out, though.